### ISSUE TITLE
Make it clear to users how to get to CLI/query service as alternative to using the Data Browser

### ISSUE THREAD

**Issue by adriennes** [DataBiosphere/data-portal#507]
For a user utilizing a screen reader (or someone who just prefers to use a CLI to the data browser), we should make it very clear that there is an alternative to using the data browser.  There should be a short redirect link somewhere on the data browser page.

**Comment by NoopDog** [comment]
closing CLI is being sunset
